app/models/business.rb

```diff
@@ -1,12 +1,16 @@
 class Business < ApplicationRecord
   include Avatarable
 
+  enum :new_developer_notifications, [:no, :daily, :weekly], default: :no,
+    suffix: :developer_notifications
+
   belongs_to :user
   has_many :conversations, -> { visible }
 
   validates :name, presence: true
   validates :company, presence: true
   validates :bio, presence: true
+  validates :new_developer_notifications, presence: true
```

Comment:
I think we can avoid an inclusion validation here. Originally, I tried adding one, but it gets weird with how enumerables work. Additionally, the enumerable will through an argument air if it's given weird. Do we need to do something to catch those errors?